__tests__/app.test.js

```diff
@@ -132,6 +132,65 @@ describe("GET /api/articles/:article_id/comments", () => {
     })
 })
 
+describe("POST /api/articles/:article_id/comments", () => {
+    test("201 - inserts a new comment to an article in the database", () => {
+        const newComment = {
+            username: "lurker",
+            body: "imo mitch isn't even that good"
+        }
+        return request(app)
+            .post("/api/articles/2/comments")
+            .send(newComment)
+            .expect(201)
+            .then(({ body }) => {
+                expect(body.comment[0].comment_id).toBe(19)
+                expect(body.comment[0].votes).toBe(0)
+                expect(typeof body.comment[0].created_at).toBe("string")
+                expect(body.comment[0].author).toBe("lurker")
+                expect(body.comment[0].body).toBe("imo mitch isn't even that good")
+                expect(body.comment[0].article_id).toBe(2)
```

Comment:
Nice! You could also use the jest matcher `.toMatchObject` to catch all these in one assertion.